apex_launchtest/test/test_io_handler_and_assertions.py

```diff
@@ -31,23 +48,24 @@ def setUpClass(cls):
 
         cls.proc_output = ActiveIoHandler()
 
-        cls.node_1 = Node(package='apex_rostest',
-                          node_executable='terminating_node',
-                          env=node_env)
+        cls.node_1 = launch.actions.ExecuteProcess(
```

Comment:
@pbaughman still a few `node`s laying around :)

Reply:
:roll_eyes: on it!

Reply:
@hidmic done.  Node has been changed to 'proc' if it was source code, or 'process' if it was a description in a doc comment